config/settings/production.py

```diff
@@ -13,7 +13,7 @@
 ALLOWED_HOSTS = env.str("ALLOWED_HOSTS").split(",")
 
 # Configure Sentry Logging
-SENTRY_DSN = env.str("SENTRY_DSN", "")
+SENTRY_SDSN = env.str("SENTRY_DSN", "")
```

Comment:
This shouldn't be changed (or rather, it can be removed since it's now declared in `base.py`)